src/server/scripts/EasternKingdoms/BlackrockMountain/MoltenCore/boss_magmadar.cpp

```diff
@@ -109,142 +138,64 @@ class boss_magmadar : public CreatureScript
     }
 };
 
-// Serrated Bites timer may be wrong
-class npc_magmadar_core_hound : public CreatureScript
```

Comment:
Is it intended to remove this script? If so remove it in DB too

Reply:
Yes, the corehound script got moved into its own script but wanted to make a seperate PR for that.

Reply:
You have 2 possibilities:

1. Keep the code and make a new PR to remove and rework to SAI
2. Provide the SAI in this PR.

Don't remove this here and fix it in another PR as it would break the creature until the ither PR was merged.

Reply:
There is not much I can do about it, normally it was one big PR I am just picking it apart here for better reviewability. 
The corehound script is still C++ at this point its just moved to its own file. I guess those 2 PRs would be merged one after another anyway.

If Yehonal is okay with it I will convert this issue into one for Magmadar + Core hounds so nothing gets broken while merging.

Reply:
If you add the fix for corehound then there's no issue removing it here